app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
-end
+    protect_from_forgery with: :exception
+
+    before_action :configure_permitted_parameters, if: :devise_controller?
+
+    protected
+
+    def configure_permitted_parameters
+        devise_parameter_sanitizer.permit(:sign_up)
```

Comment:
why have you added this?

Reply:
we are going to need it later when we implement cartify with more parameters ("name" for example).

Reply:
Premature optimization. Need to stay in scope. Please remove it if it has nothing to do with the feature

Reply:
removed.